crates/ruff_server/src/server.rs

```diff
@@ -74,6 +78,14 @@ impl Server {
                 .unwrap_or_else(|| serde_json::Value::Object(serde_json::Map::default())),
         );
 
+        crate::trace::init_tracing(
+            connection.make_sender(),
+            global_settings
+                .log_level()
+                .unwrap_or(crate::trace::LogLevel::Info),
+            global_settings.log_file(),
+        );
```

Comment:
Is it required to restart the server when changing the `logFile` or `logLevel` setting?

Reply:
Yes, that's how its set up right now.